readme.md

```diff
@@ -29,7 +29,8 @@ getPort({port: 3000}).then(port => {
 	// Will use 3000 if available, otherwise fall back to a random port
 });
 ```
-
+## Limitations
+-  In practice, under typical circumstances, the port number almost always will be available to use. But, there's a possibility of race condition as another service may have started to occupy the port in between the time it takes to determine whether the port is available and the program actually starts using the port. So, It isn't always guaranteed to be available. 
```

Comment:
Yes, that's the info I wanted, but I think it could be written more succinctly.

Reply:
Sure, I will make it more clear.